src/Frontend/src/components/failedmessages/DeletedMessageGroups.vue

```diff
@@ -68,12 +67,12 @@ function saveDefaultGroupingClassifier(classifier: string) {
   cookies.set("archived_groups_classification", classifier);
 }
 
-function classifierChanged(classifier: string) {
+async function classifierChanged(classifier: string) {
   saveDefaultGroupingClassifier(classifier);
 
   selectedClassifier.value = classifier;
   archiveGroups.value = [];
-  messageGroupList.value = loadArchivedMessageGroups(classifier);
```

Comment:
`messageGroupList` was not being used as a `ref` nor for anything given that `loadArchivedMessageGroups(classifier)` returns void!